fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetadata.java

```diff
@@ -165,6 +169,11 @@ public Statistics getTableStatistics(OptimizerContext session,
         return statistics;
     }
 
+    @Override
+    public HdfsEnvironment getHdfsEnvironment() {
+        return hdfsEnvironment;
```

Comment:
Why does every class have getHdfsEnv(). I think it should converge

Reply:
Because HiveMetadata is implemented `ConnectorMetadata`, I think we can't converge it.